src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -49,6 +50,40 @@ public void DefaultToString(double number, int innerIterations)
             }
         }
 
+        [Benchmark]
+        [InlineData("-∞", 10_000_000)]                      // Negative Infinity
+        [InlineData("-1.7976931348623157E+308", 100_000)]   // Min Negative Normal
```

Comment:
I'm not sure how much it matters, but do we want numbers that are larger than `10`?

Reply:
That shouldn't matter as much. The current algorithm is mostly dependent on the length of the input string rather than the actual magnitude of its value.